setup.py

```diff
@@ -59,7 +59,7 @@
     long_description=long_description,
     long_description_content_type="text/markdown",
     install_requires=install_requires,
-    python_requires=">=3.7, <4",
+    python_requires=">=3.7, <=3.10",
```

Comment:
Shouldn't this be

```suggestion
    python_requires=">=3.7, <=3.9",
```

?

Reply:
True.